00002_arithmetic.js

```diff
@@ -0,0 +1,49 @@
+/*2. Enter 2 numbers and display the result for the following operations (each operation should be a function)
+    add / multiply / divide / modulus / substract / greater than / less than / is equal.
+*/
+
+const readLine = require("readline-sync");
+const number1 = readLine.question("Enter first Number : ");
+const number2 = readLine.question("Enter the second Number: ");
+
+let addition = (number1, number2) => {
+    let result = parseInt(number1) + parseInt(number2);
+    console.log("Addition:", number1, "+", number2, "=", result);
+}
+
+let subtraction = (number1, number2) => {
+    let result = parseInt(number1) - parseInt(number2);
+    console.log("Subtraction:", number1, "-", number2, "=", result);
+}
+
+let multiplication = (number1, number2) => {
+    let result = parseInt(number1) * parseInt(number2);
+    console.log("Multiplication:", number1, "*", number2, "=", result);
+}
+
+let division = (number1, number2) => {
+    let result = parseInt(number1) / parseInt(number2);
+    console.log("Division:", number1, "/", number2, "=", result);
+}
+
+let modulus = (number1, number2) => {
+    let result = parseInt(number1) % parseInt(number2);
+    console.log("Modulus:", number1, "%", number2, "=", result);
+}
+
+let numberCompare = (number1, number2) => {
```

Comment:
Return values instead of printing it

Reply:
Ok. Console removed in every function  and returned the results.